packages/compass-indexes/src/utils/parse-error.ts

```diff
@@ -0,0 +1,21 @@
+import { MongoError } from 'mongodb';
+import type { AnyError } from 'mongodb';
+
+export type IndexesError = AnyError | string;
+/**
+ * Data Service attaches string message property for some errors, but not all
+ * that can happen during index creation/dropping.
+ * Check for data service custom error, then node driver errmsg.
+ */
```

Comment:
removed

Reply:
I meant more like the whole function, not only comment because it's literally what data service methods are doing when re-throwing errors coming from driver:

https://github.com/mongodb-js/compass/blob/6273cb62f7bb18907dcc8cdef59ce8ec83dc9d7a/packages/data-service/src/data-service.ts#L742-L749

(it's fine anyway though, just wanted to clarify what I meant)

Reply:
thanks for sharing. yes, i'll remove it